pandas/core/indexes/base.py

```diff
@@ -2788,26 +2788,55 @@ def set_value(self, arr, key, value):
 
     def _get_level_values(self, level):
         """
+        Return an Index of values for requested level.
+
         Return an Index of values for requested level, equal to the length
         of the index.
 
         Parameters
         ----------
         level : int or str
-            ``level`` is either the integer position of the level in the
-            MultiIndex, or the name of the level.
+            It is either the integer position of the level in the
+            MultiIndex, or the `name` of the level.
+            If given as an integer, it must be between 0 and the
+            number of levels.
 
         Returns
         -------
         values : Index
-            ``self``, as there is only one level in the Index.
+            `self`, as there is only one level in the Index.
 
         See also
         ---------
         pandas.MultiIndex.get_level_values : get values for a level of a
                                              MultiIndex
-        """
 
+        Notes
+        ---------
+        For `Index`, level should be 0, since there are no multiple levels.
+
+        Examples
+        ---------
+
+        Create an Index:
+
+        >>> idx = pd.Index(list('abc'))
+
+        Get level value by supplying level as integer:
+
+        >>> idx.get_level_values(0)
+        Index(['a', 'b', 'c'], dtype='object')
+
+        Create a MultiIndex:
+
+        >>> mi = pd.MultiIndex.from_arrays((list('abc'), list('def')))
```

Comment:
can pass `names=` to `from_arrays`

Reply:
Although line wrapping would be needed and this case and it would still end up in two lines, which isn't as nice in my opinion. Although I'll change it as you've requested.